src/encord_active/lib/project/project_file_structure.py

```diff
@@ -2,7 +2,8 @@
 from pathlib import Path
 from typing import Iterator, NamedTuple, Optional, Union
 
-from encord_active.lib.db.__autogen__ import models
+from prisma import models
```

Comment:
probably we should add prisma initialization before importing the models, or moving this import to after the prisma db connection import

Reply:
Good point, I'll move the try/except block to the `db` module init